config_test.go

```diff
@@ -672,6 +674,68 @@ func TestLoadConfig(t *testing.T) {
 	}
 }
 
+func TestGetEnvFunction(t *testing.T) {
```

Comment:
this test needs to test env with 0 variables set.

unset all variables then see what it returns.

Reply:
But then we will only be able to view the variable and not test it

Reply:
I meant in addition to what it already does. Or add a 2nd test for that scenario